myst_nbs/diagnostics_and_criticism/model_averaging.myst.md

```diff
@@ -223,12 +232,25 @@ papermill:
   status: completed
 tags: []
 ---
-az.plot_density(traces, var_names=["alpha", "sigma"]);
+ax = az.plot_density(
+    traces,
+    var_names=["alpha", "sigma"],
+    shade=0.1,
+    data_labels=["Model 0 (neocortex)", "Model 1 (log_mass)", "Model 2 (neocortex+log_mass)"],
+)
+
+ax[0, 0].set_xlabel("Density")
+ax[0, 0].set_ylabel("")
+ax[0, 0].set_title("95% Credible Intervals: alpha")
+
+ax[0, 1].set_xlabel("Density")
+ax[0, 1].set_ylabel("")
+ax[0, 1].set_title("95% Credible Intervals: sigma")
 ```
 
 +++ {"papermill": {"duration": 0.055089, "end_time": "2020-11-29T12:14:57.977616", "exception": false, "start_time": "2020-11-29T12:14:57.922527", "status": "completed"}, "tags": []}
 
-Now that we have sampled the posterior for the 3 models, we are going to use WAIC (Widely applicable information criterion) to compare the 3 models. We can do this using the `compare` function included with PyMC3.
+Now that we have sampled the posterior for the 3 models, we are going to use WAIC (Widely applicable information criterion) to compare the 3 models. We can do this using the `compare` function included with PyMC.
```

Comment:
similar comment to the one in the introduction, `compare` is an arviz function now. 

We should probably also add a note or comment on the code. I think the code does use waic, but `az.compare` now defaults to using loo instead, so running the same code will not use waic anymore.

Reply:
or maybe leave a note on the issue for whoever updates the code and reruns it on pymc v4? (which can't really be done for now as sample_posterior_predictive_w doesn't work on v4 yet)